modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/js/SampleWalkthrough.js

```diff
@@ -12,47 +12,61 @@
  * details.
  */
 
-import {Walkthrough} from 'frontend-js-components-web';
+import Walkthrough from '@liferay/frontend-js-walkthrough-web/Walkthrough';
 import React from 'react';
 
 const WALKTHROUGH_CONFIG = {
 	closeOnClickOutside: false,
 	closeable: true,
+	pages: {
+		[location.pathname + location.search + location.hash]: [
```

Comment:
Hey @matuzalemsteles, could you add `themeDisplay.getPathContext()` before `location.pathname`? This should work on DXP instances with different `pathContext`. @dsanz, this example could be another interesting for creating a FE AbsolutePortalUrlBuilder(?)

Reply:
oh yeah, well remembered

Reply:
good catch. We have a neat API Iván created recently. Perhaps we can attempt to wire it here even if we have to tweak the sample from a JSP to grab the right info